src/WebJobs.Script.WebHost/Security/Authentication/Jwt/ScriptJwtBearerExtensions.cs

```diff
@@ -65,17 +72,23 @@ public static AuthenticationBuilder AddScriptJwtBearer(this AuthenticationBuilde
 
         private static TokenValidationParameters CreateTokenValidationParameters()
         {
-            string defaultKey = Util.GetDefaultKeyValue();
-
             var result = new TokenValidationParameters();
-            if (defaultKey != null)
+            if (SecretsUtility.TryGetEncryptionKey(out byte[] key))
```

Comment:
another potential perf optimization for a later PR: can this key be cached? Do we expect it to change over the lifetime of an app?

Reply:
Yes, I thought of the same thing as well, but didn't want to include that here. Yes, we could have been caching this all along for the lifetime of a host. We can do that in the future.